packages/styling/src/interfaces/IFontStyles.ts

```diff
@@ -13,7 +13,17 @@ export interface IFontStyles {
   mediumPlus: IRawStyle;
   large: IRawStyle;
   xLarge: IRawStyle;
+  /**
+   * @deprecated Exists for forward compatibility with Fabric 7's Fluent theme.
```

Comment:
what's the backwards version of deprecated?

Reply:
@phkuo I think @detacerped would do the trick 😄

Reply:
> @phkuo I think @detacerped would do the trick 😄

lol